### PR TITLE
Feat/end user/layout

### PR DESCRIPTION
**Pull request by VanHoanTran**
I just make the Appbar work correctly and a simple profile update. I think we need to get the user's appointments into their profile also.

### CODE REVIEW

**Comment by VanHoanTran** [comment]
close #108

File: packages/end-user-fe/src/components/EndUserLayout/EndUserAppBar/index.js [hunk 1]
```diff
@@ -23,26 +23,28 @@ import {
...
 
   const shopDetails = useSelector(getShopSettings);
   const { name: shopName, logo } = shopDetails;
-  const dispatch = useDispatch();
-  const navigate = useNavigate();
 
   const onLogout = useCallback(() => {
     dispatch(removeToken());
-    setAnchorElUser(null);
+    dispatch(fetchShopSettings());
```
Comment (ghost) [inline packages/end-user-fe/src/components/EndUserLayout/EndUserAppBar/index.js]: What is the reason for fetching the shop details here?

Comment (ghost) [reply packages/end-user-fe/src/components/EndUserLayout/EndUserAppBar/index.js]: Are we losing the shop data on logout?

If this is the case, in the reducer you need to remove the purge line on the reducer that stores the shop settings

**Review by ghost — CHANGES_REQUESTED** [review]

File: packages/end-user-fe/src/components/EndUserLayout/EndUserAppBar/index.js (hunk 1 above)
Comment (VanHoanTran) [reply packages/end-user-fe/src/components/EndUserLayout/EndUserAppBar/index.js]: Thank you. That was what I want. I remove the `purgeActionTypes` in the `queryReducer`

**Review by ghost — APPROVED** [review]

**Pull request merged**